python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -112,6 +115,11 @@ router bgp {{ router_bgp.as }}
 {%     if router_bgp.bgp.bestpath.d_path is arista.avd.defined(true) %}
    bgp bestpath d-path
 {%     endif %}
+{%     if router_bgp.send_community is arista.avd.defined('all') %}
+   neighbor default send-community
+{%     elif router_bgp.send_community is arista.avd.defined %}
+   neighbor default send-community {{ router_bgp.send_community }}
```

Comment:
This should ring all alarm bells. The data model is not similar to the CLI. Please update the data model to be closer to the cli. I think we have other neighbor default commands in some address families which may (or may not) serve as inspiration.

Reply:
Thanks, fixed it!